src/sunpy_sphinx_theme/__init__.py

```diff
@@ -58,6 +58,7 @@ def default_navbar():
         ("Get Help", "help/", 2),
         ("Contribute", "contribute/", 2),
         ("Blog", "blog/", 2),
+        ("Citing SunPy", "https://docs.sunpy.org/en/stable/citation.html", 3),
```

Comment:
I'm a little reluctant to add more stuff to the very top level of the navbar, there's a practical limit to the number of things we can have before the nav bar starts looking wrong on certain screen widths.

Reply:
i totally get that - but I think having it prominent across the templates would be good?

Reply:
or any other suggestion about where to put it?

Reply:
We could move affiliated packages under about?!

Reply:
hmm i dont know, its also good to have that on the landing page with an easy button?

Reply:
yes, this happens below 1082 pixels wide:

![Screen Shot 2025-03-27 at 15 49 51](https://github.com/user-attachments/assets/8a043757-4470-415f-97ec-5481b5cadf51)

Reply:
We can attempt to fix this bug rather than work around it with less things of course.

Reply:
ugh yeah thats gross.